rustfmt.toml

```diff
@@ -0,0 +1,26 @@
+# stable
+newline_style = "Unix"
+hard_tabs = false
+tab_spaces = 4
+# blank_lines_upper_bound = 0
+# blank_lines_lower_bound = 0
+# fn_args_layout = "Vertical"
+# force_multiline_blocks = true
+# imports_layout = "Vertical"
+# imports_granularity = "Item"
+# reorder_impl_items= true
+# reorder_modules = true
+# report_todo= "Always"
+# report_fixme = "Always"
```

Comment:
these two seem to only exist in rustfmt 1.4. Are we using 1.4? why not 1.5? @apollo-pablo

Reply:
they were removed as the authors didn't think they belonged in rustfmt.